app/views/research_sessions/_share.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="share">
+  <div class="share__button">
+    <button class="button button--small button--green button--no-margin js-share-trigger">Share</button>
+  </div>
+  <div class="share__content js-share-target">
+    <p>Here is the link you can share and use at any time:</p>
+
+    <div class="share__url">
+      <input
+        type="text"
+        id="share-input"
+        class="share__input"
+        value="<%= request.original_url %>"
+        readonly />
+
+      <button class="button button--no-margin button--copy js-copy-url-button" data-clipboard-target="#share-input">Copy</button>
+    </div>
+
+    <p><a href="mailto:?subject=<%= URI.escape(@research_session.name) %>&body=<%= URI.escape(request.original_url) %>" target="_blank">You can email this link by clicking here</a></p>
+    <p>You can add this page to your bookmarks by pressing:</p>
+    <ul class="bullet-point-list">
+      <li>Command/Cmd + D on Mac</li>
+      <li>CTRL + D on Windows</li>
+    </ul>
+  </div>
+  <div class="share__overlay js-share-target js-share-trigger"></div>
+</div>
```

Comment:
Should probably place all this copy in `en.yml`

Reply:
I wouldn't worry about it for now – if it comes to real I18n later, we'd probably [internationalise the whole partial](http://guides.rubyonrails.org/i18n.html#localized-views) rather than breaking things up into copy chunks and finding some YAML home for them.

We are getting other benefits from the I18n of labels, chiefly the centralisation of both label and ActiveRecord validation messages to point to the same thing.

Reply:
(that said there's also the [safe HTML](http://guides.rubyonrails.org/i18n.html#using-safe-html-translations) option)